File: specification/vmware/resource-manager/Microsoft.AVS/stable/2021-12-01/examples-js/PrivateClouds_RotateNsxtPassword.js
const { AzureVMwareSolutionAPI } = require("@azure/arm-avs");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Rotate the NSX-T Manager password
 *
 * @summary Rotate the NSX-T Manager password
 * x-ms-original-file: specification/vmware/resource-manager/Microsoft.AVS/stable/2021-12-01/examples/PrivateClouds_RotateNsxtPassword.json
 */
async function privateCloudsRotateNsxtPassword() {
  const subscriptionId = "{subscription-id}";
  const resourceGroupName = "group1";
  const privateCloudName = "cloud1";
  const credential = new DefaultAzureCredential();
  const client = new AzureVMwareSolutionAPI(credential, subscriptionId);
  const result = await client.privateClouds.beginRotateNsxtPasswordAndWait(
    resourceGroupName,
    privateCloudName
  );
  console.log(result);
}

privateCloudsRotateNsxtPassword().catch(console.error);

File: specification/vmware/resource-manager/Microsoft.AVS/stable/2022-05-01/examples-java/PrivateClouds_RotateNsxtPassword.java
import com.azure.core.util.Context;

/** Samples for PrivateClouds RotateNsxtPassword. */
public final class Main {
    /*
     * x-ms-original-file: specification/vmware/resource-manager/Microsoft.AVS/stable/2022-05-01/examples/PrivateClouds_RotateNsxtPassword.json
     */
    /**
     * Sample code: PrivateClouds_RotateNsxtPassword.
     *
     * @param manager Entry point to AvsManager.
     */
    public static void privateCloudsRotateNsxtPassword(com.azure.resourcemanager.avs.AvsManager manager) {
        manager.privateClouds().rotateNsxtPassword("group1", "cloud1", Context.NONE);
    }
}
